app/src/main/kotlin/dev/soupslurpr/appverifier/InternalVerificationInfoDatabase.kt

```diff
@@ -1154,4 +1154,164 @@ val internalVerificationInfoDatabase = setOf(
+    ),
+    InternalDatabaseVerificationInfo(
+        "ru.zdevs.zugate",
+        listOf(
+            // Google Play Store
+            Hashes(
+                listOf(
+                    "74:DD:42:51:7D:0A:30:DB:4E:13:1D:C3:FF:D1:CF:11:D0:FD:95:75:3C:7D:27:C3:EB:33:2A:E0:59:92:DD:0E"
+                ),
+                false
+            )
+        )
+    ),
+    InternalDatabaseVerificationInfo(
+        "org.connectbot",
+        listOf(
+            // GitHub
+            Hashes(
+                listOf(
+                    "AA:43:74:6A:46:12:78:42:16:9F:81:76:41:2A:1F:4B:1B:1E:2F:8A:D9:E1:5A:BA:21:7C:38:22:DB:EB:2D:5F"
+                ),
+                false
+            ),
+            // Google Play Store
+            Hashes(
+                listOf(
+                    "AA:43:74:6A:46:12:78:42:16:9F:81:76:41:2A:1F:4B:1B:1E:2F:8A:D9:E1:5A:BA:21:7C:38:22:DB:EB:2D:5F",
+                    "2F:EF:9C:FE:29:BE:E1:93:20:61:A8:1B:FB:B8:B5:7C:5C:65:88:54:B8:2C:C9:7B:BF:26:52:65:DD:B2:27:7C"
+                ),
+                false
+            ),
+            // F-Droid
+            Hashes(
+                listOf(
+                    "08:78:9B:AD:18:CE:8E:C7:B6:63:7B:5E:70:24:5A:76:3A:E5:02:4F:4D:49:AC:85:32:40:47:FB:06:B5:DA:8A"
+                ),
+                false
+            )
+        )
+    ),
+    InternalDatabaseVerificationInfo(
+        "eu.depau.etchdroid",
+        listOf(
+            // GitHub
+            Hashes(
+                listOf(
+                    "B4:F8:85:40:B9:2C:42:15:1F:0F:8A:66:97:7D:1E:32:54:37:9F:4E:97:45:A4:9E:0C:85:B0:F6:44:B1:82:81"
+                ),
+                false
+            ),
+            // Google Play Store
+            Hashes(
+                listOf(
+                    "B2:42:E9:89:E4:01:36:6D:0E:92:4B:15:C9:BC:66:76:7D:A7:4A:B3:08:FF:21:B0:BA:50:33:02:81:59:A6:AE"
+                ),
+                false
+            ),
+            // F-Droid
+            Hashes(
+                listOf(
+                    "BB:B3:C5:88:55:98:00:54:DF:31:5B:F5:1C:04:5F:F2:B5:01:C5:87:06:51:14:59:FC:34:53:27:DF:BF:3E:CE"
+                ),
+                false
+            )
+        )
+    ),
+    InternalDatabaseVerificationInfo(
+        "chat.simplex.app",
+        listOf(
+            // GitHub
+            Hashes(
+                listOf(
+                    "3C:52:C4:FD:3C:AD:1C:07:C9:B0:0A:70:80:E3:58:FA:B9:FE:FC:B8:AF:5A:EC:14:77:65:F1:6D:0F:21:AD:85"
+                ),
+                false
+            ),
+            // Google Play Store
+            Hashes(
+                listOf(
+                    "5E:3E:DC:C2:00:FB:A8:D5:F4:88:F3:CA:4C:32:5B:05:78:C5:6A:9C:03:A1:CC:B5:92:9C:D7:5C:7E:57:E2:4D"
+                ),
+                false
+            ),
+            // SimpleX Chat F-Droid
+            Hashes(
+                listOf(
+                    "9F:35:8F:F2:84:D1:F7:16:56:A2:BF:AF:0E:00:5D:EA:E6:AA:14:14:37:20:E0:89:F1:1F:F2:DD:CF:EB:01:BA"
```

Comment:
Can you check again? For me its 
chat.simplex.app
3C:52:C4:FD:3C:AD:1C:07:C9:B0:0A:70:80:E3:58:FA:B9:FE:FC:B8:AF:5A:EC:14:77:65:F1:6D:0F:21:AD:85